examples/storm-perf/pom.xml

```diff
@@ -22,8 +22,8 @@
     <parent>
         <artifactId>storm</artifactId>
         <groupId>org.apache.storm</groupId>
-        <version>1.1.1-SNAPSHOT</version>
-        <relativePath>..</relativePath>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
Maybe `../../pom.xml`?
(given that storm-perf compile is failing in Travis build... https://travis-ci.org/apache/storm/jobs/212268892)